### ISSUE TITLE
应该在训练前检查是否有权重，有则加载，否则一不小心白训练了

### ISSUE THREAD

**Issue by aaastar** [jingyaogong/minimind#257]
ckp_path = f'{args.save_dir}/pretrain_{lm_config.dim}.pth'
    if os.path.exists(ckp_path):
        Logger(f"发现已有权重文件 {ckp_path}，加载中...")
        state_dict = torch.load(ckp_path, map_location=args.device)
        if isinstance(model, torch.nn.parallel.DistributedDataParallel):
            model.module.load_state_dict(state_dict)
        else:
            model.load_state_dict(state_dict)
        Logger("权重加载完成，继续训练")
    else:
        Logger("未发现已有权重文件，从头开始训练")
